cheroot/server.py

```diff
@@ -1523,6 +1523,11 @@ class HTTPServer:
     timeout = 10
     """The timeout in seconds for accepted connections (default 10)."""
 
+    expiration_interval = 0.5
+    """The interval, in seconds, at which the server checks for
+    expired connections (default 0.5).
```

Comment:
a bit arbitrary, but this matches the [socketserver default](https://docs.python.org/3/library/socketserver.html#socketserver.BaseServer.serve_forever)